department-of-veterans-affairs/abd-vro
# Make parallel call lighthouse fhir api.

Call Lighthouse FHIR API in parallel

## What was the problem?
Changing the sequential calls to Lighthouse FHIR API for Blood Pressure, Condition, and Medication to parallel calls to improve the VRO response time.

Associated tickets or Slack threads:
MCP-2526

## How does this fix it?[^1]
Change sequential calls to parallel.

## How to test this PR
- Step 1
- Step 2[^secrel]


Note: The Lighthouse Sandbox account used in local environment has a connection limit. We may need to confirm this before this PR got merged.

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
